lib/ddtrace/auto_instrument.rb

```diff
@@ -0,0 +1,3 @@
+require 'ddtrace'
+
+Datadog.add_auto_instrument if Datadog.respond_to?(:add_auto_instrument)
```

Comment:
`Datadog.add_auto_instrument` is always defined now, as of latest changes, right?

Reply:
yup, can remove the defensive check here now